test_util/marathon.py

```diff
@@ -214,7 +293,7 @@ def destroy_app(self, app_name, timeout=120):
         @retrying.retry(wait_fixed=1000, stop_max_delay=timeout * 1000,
                         retry_on_result=lambda ret: not ret,
                         retry_on_exception=lambda x: False)
-        def _destroy_complete(deployment_id):
+        def _destroy_pod_complete(deployment_id):
```

Comment:
wow... I'm not sure what happened there.  this was resolved and the correct method name refactored.